docs/dqx/docs/installation.mdx

```diff
@@ -25,12 +28,12 @@ Install a specific version of the project via `pip` (e.g. version 0.1.12):
 pip install databricks-labs-dqx==0.1.12
 ```
 
-## DQX installation in a Databricks Workspace
+## DQX installation as Tool in a Databricks Workspace
```

Comment:
"as Tool" -> "as a Tool"? 

Maybe we need to run everything through Grammarly?